x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_landing/integration_assistant_card.tsx

```diff
@@ -63,9 +38,27 @@ export const IntegrationAssistantCard = React.memo(() => {
             justifyContent="flexStart"
           >
             <EuiFlexItem>
-              <EuiTitle size="xs">
-                <h3>{i18n.ASSISTANT_TITLE}</h3>
-              </EuiTitle>
+              <EuiFlexGroup
+                direction="row"
+                gutterSize="s"
+                // alignItems="flexStart"
+                // justifyContent="flexStart"
```

Comment:
Can we remove them if they are not required?

Reply:
Ops, yeah I forgot it